dissect-netwayste/src/netwaysteparser.rs

```diff
@@ -282,3 +290,321 @@ pub fn parse_netwayste_format() -> HashMap<CString, NetwaysteDataFormat> {
 
     map
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[test]
+    fn test_parse_size_from_type_for_u8() {
+        let string = "u8".to_owned();
+        let expected_sizing = vec![Sizing::Fixed(1)];
+
+        let parsed_sizing = parse_size_from_type(string);
```

Comment:
I like the test coverage in this file!

Can you please factor out this comparison code into a function though?

Reply:
Aye!